seshdash/views.py

```diff
@@ -724,16 +724,31 @@ def historical_data(request):
 #function for Graph Generations   
 @login_required
 def graphs(request):
+
+    # if ajax request
     if request.method == 'POST':
+
+        # variables declaration
         results = {}
+        time_delta_dict = {}
+        time_bucket_dict = {}
         data_values =[]
         date_time = []
+
+        # Getting values from Post request
         time = request.POST.get('time','')
         choices = request.POST.getlist('choice[]')
         active_id = request.POST.get('active_site_id','')
         active_site = Sesh_Site.objects.filter(id=active_id)
-        active_site_name = active_site [0]
+       
+        # Checking for a valid site_id
+        if active_site != []:
```

Comment:
in python you can simply say `if active_site` this will return False if the list is empty